Objective-C/Tests/ReplicatorTest+CustomConflict.m

```diff
@@ -418,6 +419,60 @@ - (void) _testConflictResolverThrowingException {
     AssertEqualObjects([self.db documentWithID: docId].toDictionary, remoteData);
 }
 
+- (void) testConflictResolutionDefault {
+    NSError* error;
+    NSDictionary* localData = @{@"key1": @"value1"};
+    NSDictionary* remoteData = @{@"key2": @"value2"};
+    NSMutableArray* conflictedDocs = [NSMutableArray array];
+    
+    // higher generation-id
+    NSString* docID = @"doc1";
+    [self makeConflictFor: docID withLocal: localData withRemote: remoteData];
+    CBLMutableDocument* doc = [[self.db documentWithID: docID] toMutable];
+    [doc setValue: @"value3" forKey: @"key3"];
+    [self saveDocument: doc];
+    [conflictedDocs addObject: @[[self.db documentWithID: docID],
+                                 [otherDB documentWithID: docID]]];
+    
+    // delete local
+    docID = @"doc2";
+    [self makeConflictFor: docID withLocal: localData withRemote: remoteData];
+    [self.db deleteDocument: [self.db documentWithID: docID] error: &error];
+    [conflictedDocs addObject: @[[self.db documentWithID: docID] ?: [NSNull null],
+                                 [otherDB documentWithID: docID]]];
+    
+    // delete remote
+    docID = @"doc4";
+    [self makeConflictFor: docID withLocal: localData withRemote: remoteData];
+    [otherDB deleteDocument: [otherDB documentWithID: docID] error: &error];
+    [conflictedDocs addObject: @[[self.db documentWithID: docID],
+                                 [otherDB documentWithID: docID] ?: [NSNull null]]];
```

Comment:
Is there a case that [otherDB documentWithID: docID] is not null here?

Reply:
same. we can directly assign null.

Reply:
```suggestion
                                 [NSNull null]]];
```